### PR TITLE
Game repo

### LINKED ISSUE #30: game_repo
Setup file structure for the Game.rb repo

### PR DESCRIPTION
**Pull request by b-enji-cmd**

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -10,17 +10,27 @@ class Game
...
-    @season = season
-    @type = type
-    @date_time = date_time
-    @away_team_id = away_team_id
-    @home_team_id = home_team_id
-    @away_goals = away_goals
-    @home_goals = home_goals
-    @venue = venue
-    @venue_link = venue_link
+  def calculate_winner
```
Comment (aetzion1) [inline lib/game.rb:26]: Why is this helper method here vs. in the repo? What should we do moving forward re: helper methods? (Adam & Isabelle)

**Review by aetzion1 — APPROVED** [review]
Looks great! Let's talk through whether we should have helper methods in the game/game_teams/team.rb files or only in the repo files.

File: lib/game.rb (hunk 1 above)
Comment (isabellevillasenor) [reply lib/game.rb:26]: I second that! ^ We can talk tmrw at 1 about it (:

**Review by isabellevillasenor — APPROVED** [review]
Awesome work friends!

**Pull request merged**